src/views/SpaceProposals.vue

```diff
@@ -159,10 +159,7 @@ onMounted(() => {
             :space="space"
             :voted="userVotedProposalIds.includes(proposal.id)"
             :hide-space-avatar="proposal.space.id === space.id"
-            :to="{
-              name: 'spaceProposal',
-              params: { id: proposal.id }
-            }"
+            :to="{ path: `/${proposal.space.id}/proposal/${proposal.id}` }"
```

Comment:
This will not work for custom domains. You can try by adding `"localhost": "samuv.eth"` to `snapshot-spaces/spaces/domains.json` in local environment.

Reply:
You can check for custom domain using `useApp` and `domain`